### DISCUSSION TITLE
Do we Need to obtain Licenses of all third Party Downloads

### DISCUSSION THREAD

**Discussion by jaideep746** [ImageMagick/ImageMagick#6613] [General]
Hey Team.
Can you please elaborate and explain me that if My application is integrated with ImageMagick C++ api , Do I need to obtain license of other third party libraries that is used by ImageMagick. (for egs -Library Like heif and other such )

**Comment by urban-warrior** [comment]
The ImageMagick license pertains to the ImageMagick source code, binaries, and documentation, but it does not extend to include optional third-party delegate libraries or programs.
